components.html

```diff
@@ -0,0 +1,1094 @@
+              >
+            </li>
+            <li class="category">
+              <a href="#card-section" class="link-style" id="card">Card</a>
+            </li>
+            <li class="category">
+              <a href="#image-section" class="link-style" id="image">Image</a>
+            </li>
+            <li class="category">
+              <a href="#input-section" class="link-style" id="input">Input</a>
+            </li>
+            <li class="category">
+              <a href="#list-section" class="link-style" id="lists">Lists</a>
+            </li>
+            <li class="category">
+              <a href="#nav-section" class="link-style" id="navigation"
+                >Navigation</a
+              >
+            </li>
+            <li class="category">
+              <a href="#modal-section" class="link-style" id="modal">Modal</a>
+            </li>
+            <li class="category">
+              <a href="#rating-section" class="link-style" id="rating"
+                >Rating</a
+              >
+            </li>
+            <!-- <li class="category">
+              <a href="#" class="link-style" id="grid">Grid</a>
+            </li> -->
+            <li class="category">
+              <a href="#text-section" class="link-style" id="uitilities"
+                >Text Utilities</a
+              >
+            </li>
+            <li class="category">
+              <a href="#snackbar-section" class="link-style" id="snackbar"
+                >Snackbar</a
+              >
+            </li>
+          </ul>
+        </aside>
+        <main class="main">
+          <section id="introduction-section">
+            <h2>Introduction</h2>
+            <div class="examples">
+              <h3>Html Boiler plate to get started:</h3>
+              <div class="code-snippet">
+                <iframe
+                  src="https://carbon.now.sh/embed?bg=rgba%28231%2C236%2C234%2C1%29&t=material&wt=bw&l=htmlmixed&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=12.5px&lh=133%25&si=false&es=2x&wm=false&code=%253C%21doctype%2520html%253E%250A%250A%253Chtml%2520lang%253D%2522en%2522%253E%250A%253Chead%253E%250A%2520%2520%253Cmeta%2520charset%253D%2522utf-8%2522%253E%250A%250A%2520%2520%253Ctitle%253EThe%2520HTML5%2520Herald%253C%252Ftitle%253E%250A%2520%2520%253Cmeta%2520name%253D%2522description%2522%2520content%253D%2522The%2520HTML5%2520Herald%2522%253E%250A%2520%2520%253Cmeta%2520name%253D%2522author%2522%2520content%253D%2522SitePoint%2522%253E%250A%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Ffractalui.netlify.app%252Fcss%252Ffractal.css%2522%253E%250A%250A%253C%252Fhead%253E%250A%2520%2520%250A%253Cbody%253E%250A%2520%2520%250A%2520%2520%253Cscript%2520src%253D%2522https%253A%252F%252Funpkg.com%252Fionicons%25405.4.0%252Fdist%252Fionicons.js%2522%253E%253C%252Fscript%253E%250A%253C%252Fbody%253E%250A%253C%252Fhtml%253E"
+                  style="
```

Comment:
Can the styling on the code-snippet iframe be refactored out and placed in the css file? 

Having all the style in one place will help later when you try to add more styles. Otherwise you might have to keep 
moving back and forth between your html and css. Some of this is repeating as well.